### PR TITLE
fix: add write permission for leases

### PR DESCRIPTION
**Pull request by liodo198592**
…w_clusterrole.yaml

Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: test/e2e/scaffold/ingress.go [hunk 1]
```diff
@@ -37,7 +172,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: cluster-admin
+  name: apisix_view_clusterrole
```
Comment (gxthrj) [inline test/e2e/scaffold/ingress.go:175]: Need to keep the same as `ClusterRole` (be combined with `namespace`).

File: test/e2e/scaffold/ingress.go [hunk 2]
```diff
@@ -29,6 +29,141 @@ import (
 
 const (
 	_serviceAccount     = "ingress-apisix-e2e-test-service-account"
+	_clusterRole = `
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+	name: apisix-view-clusterrole
```
Comment (gxthrj) [inline test/e2e/scaffold/ingress.go:36]: Need to be combined with namespace.

Comment (liodo198592) [reply test/e2e/scaffold/ingress.go]: updated

**Comment by tokers** [comment]
@liodo198592 Is this PR ready for review?

File: test/e2e/scaffold/ingress.go [hunk 3]
```diff
@@ -28,7 +28,142 @@ import (
 )
 
 const (
-	_serviceAccount     = "ingress-apisix-e2e-test-service-account"
+	_serviceAccount = "ingress-apisix-e2e-test-service-account"
+	_clusterRole    = `
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (tokers) [inline test/e2e/scaffold/ingress.go:33]: The cluster role should be deleted when the case is finilazing.

**Comment by tokers** [comment]
https://github.com/apache/apisix-ingress-controller/pull/235 I have created another PR, your contributions are still in effective.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**